tests/functional/export-all-cluster-connection/source/data/config.json

```diff
@@ -4,7 +4,13 @@
       "protocol": "mongodb+srv",
       "host": "mongodb.cluster.local",
       "port": 27017,
-      "database": "test"
+      "database": "test",
+      "ssl": {
+        "enabled": true,
+        "ca": "%env(string:SSL_CA)%",
+        "cert": "%env(string:SSL_CERT)%",
+        "#key": "%env(string:SSL_KEY)%"
```

Comment:
Protokol `mongodb+srv://....` automaticky zapina `ssl=true` (da sa to najst niekde v dokumentacii).

Kedze pouzivame self-signed certifikat, tak tu musime uviest SSL konfiguraciu.

Tj. klientovy to pojde aj bez toho, ak ma normalny certifikat.